code-challenges.js

```diff
@@ -28,3 +59,36 @@ var hand2 = [5, 5, 3, 3, 4]
 // Expected output: false
 var hand3 = [5, 5, 5, 5, 4]
 // Expected output: false
+
+const pokerHand = (array) => {
+let sortedArr = array.sort()
+// hand1 now is [3, 3, 5, 5, 5]
+fullHouseArray =[]
+remaining =[]
+
+    for (let i = 0; i < sortedArr.length; i++) {
+        if (sortedArr[i + 2] == sortedArr[i+1]
+            && sortedArr[i+1] == sortedArr[i]
+            && sortedArr[i + 2] == sortedArr[i]) {
+            fullHouseArray.push(sortedArr[i+2])
+            fullHouseArray.push(sortedArr[i+1])
+            fullHouseArray.push(sortedArr[i])
+        } else {
+            remaining.push(sortedArr[i])
+        }
+    }
+    for (let j = 0; j < remaining.length-2; j++) {
+        if (remaining[j + 1] == remaining[j]) {
+            fullHouseArray.push(remaining[j])
+            fullHouseArray.push(remaining[j+1])
+        }
+    }
+    if (fullHouseArray.length==5){
+        return `NOICE, "${fullHouseArray}" is a full house!`
+    } else {
+        return "Sorry, you do NOT have a full house"
+    }
+}
```

Comment:
this is a pretty cool solution! It took my a little while to decipher this though it would've been really nice to see some pseudocode in here explaining your process. but it works! :)